### ISSUE TITLE
time series APIs are slow

### ISSUE THREAD

**Issue by hamilton** [rilldata/rill#1464]
from notion: The increased time taken on the network console is a combination of API with low priority, needing to run multiple DuckDB queries and being set run in parallel with high priority API. Setting QueryQueueSize to 1 in HttpRequestQueue shows the API’s returning in 100-200 ms.

**Comment by begelundmuller** [comment]
Will be fixed by https://github.com/rilldata/rill-developer/pull/1405, since it will ensure that all multi-query analytical APIs (i.e. analytical APIs that first create a temporary table, then query it in various ways) acquire priority once, then execute all their queries uninterrupted in sequence.
